change/@ni-nimble-components-cde0ca9f-1259-4049-ae46-ead5829d5151.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This change supercedes the need for a blank icon mapping right? Should we also revert the ability to specify an empty icon mapping? https://github.com/ni/nimble/pull/1874/files

Reply:
That's an interesting question. There is a slight difference in rendering between an empty mapping and an icon mapping without an icon. The matrix tests probably illustrate this better than I can explain, but basically, the group row reserves space for an icon on icon mappings but not for empty mappings. I'm not sure if one is better in some cases or not. I'll talk to Brandon and see if he has any thoughts on the alignment.

Reply:
Talked with @fredvisser, and he said that text-only mappings would not be recommended with icon or icon+text mappings. Therefore, I'll plan to remove the blank icon mapping and leave the alignment of everything else as-is.